tests/test_general.py

```diff
@@ -1593,18 +1593,16 @@ def test_2d_tube_intersections_bifurcation(self):
             Node.VALUE_LABEL_VALUE, Node.VALUE_LABEL_D_DS1,
             Node.VALUE_LABEL_D_DS2, Node.VALUE_LABEL_D2_DS1DS2,
             Node.VALUE_LABEL_D_DS3, Node.VALUE_LABEL_D2_DS1DS3]
-        segmentTubeData = []
+        tubeSegments = []
         trackSurfaces = []
         networkSegments = networkMesh.getNetworkSegments()
         elementsCountAround = 8
         for networkSegment in networkSegments:
             pathParameters = get_nodeset_path_ordered_field_parameters(
                 nodes, coordinates, valueLabels, networkSegment.getNodeIdentifiers(), networkSegment.getNodeVersions())
-            tubeData = SegmentTubeData(pathParameters, elementsCountAround)
-            segmentTubeData.append(tubeData)
-            rawTubeData = getPathRawTubeCoordinates(pathParameters, elementsCountAround=elementsCountAround)
-            tubeData.setRawTubeCoordinates(rawTubeData)
-            trackSurfaces.append(tubeData.getRawTrackSurface())
+            tubeSegment = TubeNetworkMeshSegment(networkSegment, [pathParameters], elementsCountAround, 1)
```

Comment:
I've added few more parameters related to the solid core to this class, so maybe it'd be better to call the arguments by name? i.e.
```suggestion
            tubeSegment = TubeNetworkMeshSegment(networkSegment, pathParametersList=[pathParameters], elementsCountAround=elementsCountAround, 
elementsCountThroughWall=1)
```